fix dockerfile, docker-compose

fizika734 opened the pull request:
FromAmazonCorrettoJdk11.Dockerfile
  fixed VOLUME instruction

docker-compose-AmazonCorretoJdk11.yml
  fixed volumes

fizika734 committed:
fix dockerfile, docker-compose

FromAmazonCorrettoJdk11.Dockerfile
  fixed VOLUME instruction

docker-compose-AmazonCorretoJdk11.yml
  fixed volumes

yilativs commented on FromAmazonCorrettoJdk11.Dockerfile:
```diff
@@ -47,4 +56,4 @@ ENV JAVA_OPTS="-Xms1g -Xmx1g"
 ENTRYPOINT ["/sbin/tini", "-v", "--", "/opt/service/entry-point.sh"]
-CMD [--spring.profiles.active=local]
\ No newline at end of file
+CMD [--spring.profiles.active=local]
```
what has changed here?

fizika734 replied:
i think this is because of the some blank lines above.

yilativs commented on FromAmazonCorrettoJdk11.Dockerfile:
```diff
@@ -16,28 +16,37 @@ FROM amazoncorretto:11-alpine-jdk
 
-RUN apk add --no-cache tini
 
 USER service:service
```
I think it should be moved before COPY,
have you checked created image with dive?

yilativs commented on FromAmazonCorrettoJdk11.Dockerfile:
```diff
@@ -16,28 +16,37 @@ FROM amazoncorretto:11-alpine-jdk
 
+# if you want to copy tini binary from builder it will not work(alpine not working with precompiled binaries from other distros)
+#https://github.com/github/hub/issues/1818
+# you should change the builder image to alpine or install tini as apk
```
let's do it.

fizika734 replied:
ok ill fix it tomorrow

yilativs reviewed (commented):
needs work:
migration tini to builder and migration builder to alpine is required.
validation with dive required.

fizika734 committed:
FromAmazonCorrettoJdk11.Dockerfile
  fixed VOLUME instruction
  base image changed to alpine
  dockerfile optimization (-20 Mb, dive image efficiency score: 99 %)
docker-compose-AmazonCorretoJdk11.yml
  fixed volumes
